troi/sorts.py

```diff
@@ -0,0 +1,34 @@
+from operator import attrgetter
+
+import troi
+from troi import PipelineError, Recording
+
+
+class YearSortElement(troi.Element):
+    '''
+        Sort recordings by year.
+    '''
+
+    def __init__(self, reverse=False):
+        '''
+            Sort recordings by year ascending. If reverse=True, sort descending.
+        '''
+        super().__init__()
+        self.reverse = reverse
+
+    @staticmethod
+    def inputs():
+        return [Recording]
+
+    @staticmethod
+    def outputs():
+        return [Recording]
+
+    def read(self, inputs):
+
+        def year_sorter(r):
+            if not r.year:
+                return 0
```

Comment:
This sorts items with no year first. Is this desired behaviour? My gut feeling is that they should be at the end.

Reply:
I didn't have a preference, so I changed it as you suggested.